sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/HierarchiesClient.cs

```diff
@@ -150,5 +152,211 @@ Page<TimeSeriesHierarchy> NextPageFunc(string nextLink, int? pageSizeHint)
                 throw;
             }
         }
+
+        /// <summary>
+        /// Gets Time Series Insights hierarchies by hierarchy names asynchronously.
+        /// </summary>
+        /// <param name="timeSeriesHierarchyNames">List of names of the Time Series hierarchies to return.</param>
```

Comment:
casing of `Time Series hierarchies` seems a bit strange to me. 
what do you think about `TimeSeries hierarchies` ? or alternatively `Time series hierarchies` ...

Reply:
This is how I see in documentation: https://docs.microsoft.com/en-us/azure/time-series-insights/concepts-model-overview#time-series-model-hierarchies

It is also a recommendation we have followed other places.